apps/backend/src/lib/projects.tsx

```diff
@@ -164,21 +150,35 @@ export function projectPrismaToCrud(
         }
       })(),
       team_creator_default_permissions: prisma.config.permissions.filter(perm => perm.isDefaultTeamCreatorPermission)
-        .map(teamPermissionDefinitionJsonFromDbType)
+        .map(permissionDefinitionJsonFromDbType)
         .concat(prisma.config.teamCreateDefaultSystemPermissions.map(db => teamPermissionDefinitionJsonFromTeamSystemDbType(db, prisma.config)))
         .sort((a, b) => stringCompare(a.id, b.id))
         .map(perm => ({ id: perm.id })),
       team_member_default_permissions: prisma.config.permissions.filter(perm => perm.isDefaultTeamMemberPermission)
-        .map(teamPermissionDefinitionJsonFromDbType)
+        .map(permissionDefinitionJsonFromDbType)
         .concat(prisma.config.teamMemberDefaultSystemPermissions.map(db => teamPermissionDefinitionJsonFromTeamSystemDbType(db, prisma.config)))
         .sort((a, b) => stringCompare(a.id, b.id))
         .map(perm => ({ id: perm.id })),
       user_default_permissions: prisma.config.permissions.filter(perm => perm.isDefaultUserPermission)
-        .map(teamPermissionDefinitionJsonFromDbType)
+        .map(permissionDefinitionJsonFromDbType)
         .sort((a, b) => stringCompare(a.id, b.id))
         .map(perm => ({ id: perm.id })),
     }
   };
+
+  const newResultWithConfigJson = renderedEnvironmentConfigToProjectCrud(dbProjectToRenderedEnvironmentConfig(prisma), result.config.id);
```

Comment:
this tests a function that we're not gonna be using in the future. Instead, you should be testing getRenderedOrganizationConfig


```suggestion
  const newResultWithConfigJson = renderedOrganizationConfigToProjectCrud(await getRenderedOrganizationConfig(), result.config.id);
```